modules/sqs/main.tf

```diff
@@ -53,17 +53,81 @@ resource "aws_sqs_queue" "metadata_queue" {
 ## ====================================================================================================
 resource "aws_sqs_queue" "staged_recovery_queue" {
   ## OPTIONAL
-  name                        = "${var.prefix}-staged-recovery-queue.fifo"
-  fifo_queue                  = true
-  content_based_deduplication = true
-  delay_seconds               = var.sqs_delay_time_seconds
-  max_message_size            = var.sqs_maximum_message_size
-  message_retention_seconds   = var.staged_recovery_queue_message_retention_time_seconds
-  tags                        = local.tags
-  policy                      = data.aws_iam_policy_document.staged_recovery_queue_policy.json
-  visibility_timeout_seconds  = 1800 # Set to double lambda max time
+  name                       = "${var.prefix}-staged-recovery-queue"
+  delay_seconds              = var.sqs_delay_time_seconds
+  max_message_size           = var.sqs_maximum_message_size
+  message_retention_seconds  = var.staged_recovery_queue_message_retention_time_seconds
+  tags                       = local.tags
+  policy                     = data.aws_iam_policy_document.staged_recovery_queue_policy.json
+  visibility_timeout_seconds = 1800 # Set to double lambda max time
+  depends_on = [
+    aws_sqs_queue.staged_recovery_deadletter_queue
```

Comment:
Should be able to remove this due to the deadletter_queue being used/depended on in the redrive_policy.

Reply:
I got circular dependency without adding this. The DLQ has to be created first before the original queue.